### ISSUE TITLE
Current command

### ISSUE THREAD

**Issue by jerone** [gabrielrobert/xavtool#10]
The current command is not consistent in returning the value. Currently it returns...
* friendly version name for Android.
* real version for iOS.

My suggestion is to make this consistent for both platforms.
And return both values (version and friendly version name).

(Windows has only one version type)

**Comment by gabrielrobert** [comment]
Example:

```
1.0.1 [1000100] - com.example.xavtool (test\AndroidManifest.xml)
1.0.1 [1.0.1] - xavtool (test\Info.plist)
1.0.1.0 [---] - xavtool (test\Package.appxmanifest)
```

Format:
`"%Version [%InternalVersion] - %Name (%Path)"`

Is it good for you, @jerone ? 

It will be part of 1.2.0. Code is here -> #16

**Comment by jerone** [comment]
I've not been able to install the development version, but what you suggest here looks great!

What about aligning the values (not sure about it)...
```
1.0.1   [1000100] - com.example.xavtool (test\AndroidManifest.xml)
1.0.1   [1.0.1]   - xavtool (test\Info.plist)
1.0.1.0 [---]     - xavtool (test\Package.appxmanifest)
```

**Comment by jerone** [comment]
Not sure if this is also fixed by #16, but I also noticed an inconsistency with `%name`; sometimes the display name is used (e.g. for iOS) and for the rest the package identifier is used.
